### PR TITLE
Explain how to check for existing PR

### PR DESCRIPTION
**Pull request by brunchboy**
To avoid extra work on the part of contributors and maintainers, this
adds a step to the explanation of how to contribute a fix to a formula,
showing how to check if someone else has already done so, even if it has
not yet finished being tested, reviewed, and released.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ To report a bug:
 
 Submit a `1.2.3` version upgrade for the `foo` formula:
 
+- First check if someone else has already submitted the same upgrade, since it can take time to be tested and incorporated. Look at the open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) and add the formula name to the filter box at the top of the list, so it looks like `is:pr is:open foo`. If something shows up, check to see if it accomplishes what you were about to work on. If nothing matched, then proceed.
```
Comment (mikemcquaid) [inline CONTRIBUTING.md]: Would really like to slim this down a lot. Maybe just point to `https://github.com/Homebrew/homebrew/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,6 +17,8 @@ To add a new formula for `foo` version `2.3.4` from `$URL`:
 
 To contribute a fix to the `foo` formula:
 
+- You might want to check to see if someone has already contributed the same fix, since it can take time to be tested and incorporated. Look at the open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) and add the formula name to the filter box at the top of the list, so it looks like `is:pr is:open foo`. If something shows up, check to see if it accomplishes what you were about to work on. If nothing matched, then proceed.
```
Comment (mikemcquaid) [inline CONTRIBUTING.md]: Less concerned about duplicate fixes; they don't really happen and if it's documented above then that's fine for me.

Comment (brunchboy) [inline CONTRIBUTING.md]: I was wondering about that. OK, I will remove this one!

File: CONTRIBUTING.md (hunk 1 above)
Comment (brunchboy) [inline CONTRIBUTING.md]: All right, I took a crack at making it much more concise. My only concern is that the “foo“ pattern matches a bunch of PRs, and I am not sure people will know to look at the filter box and change the search to match their actual formula. But I guess we can give people the benefit of the doubt?

Comment (bfontaine) [inline CONTRIBUTING.md]: You could use the actual name of a formula.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ To report a bug:
 
 Submit a `1.2.3` version upgrade for the `foo` formula:
 
+- First check if someone else has already submitted the same upgrade, since it can take time to be tested and incorporated. Search the open Pull Requests for `foo` [like this](https://github.com/Homebrew/homebrew/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo). If nothing relevant shows up, proceed.
```
Comment (mikemcquaid) [inline CONTRIBUTING.md]: Can be slimmed down a bit: `Check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).`

Comment (brunchboy) [inline CONTRIBUTING.md]: Yes, that makes it incredibly terse while even more clear. Thanks!

**Comment by mikemcquaid** [comment]
Great work here @brunchboy. Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**